subsys/bluetooth/controller/Kconfig

```diff
@@ -560,13 +560,14 @@ config BT_CTLR_LE_POWER_CONTROL
 	bool "LE Power Control Request Feature"
 	depends on BT_CTLR_LE_POWER_CONTROL_SUPPORT
 	default y if BT_TRANSMIT_POWER_CONTROL
+	default y if BT_PATH_LOSS_MONITORING
 	help
 	  Enable support for LE Power Control Request feature that is defined in the
 	  Bluetooth Core specification, Version 5.4 | Vol 6, Part B, Section 4.6.31.
 
 config BT_CTLR_LE_PATH_LOSS_MONITORING
 	bool "LE Path Loss Monitoring Feature"
-	depends on BT_CTLR_LE_PATH_LOSS_MONITORING_SUPPORT
+	depends on BT_CTLR_LE_PATH_LOSS_MONITORING_SUPPORT && BT_CTLR_LE_POWER_CONTROL
```

Comment:
What if `CONFIG_BT_CTLR=n`?

Reply:
If `CONFIG_BT_CTLR=n` then `BT_CTLR_LE_PATH_LOSS_MONITORING_SUPPORT` shouldn't be enabled